WordPress/Classes/Utility/Universal Links/Routes+Reader.swift

```diff
@@ -167,3 +163,49 @@ extension ReaderRoute: NavigationAction {
         return isYear(year) && isMonth(month) && isDay(day)
     }
 }
+
+// MARK: - RootViewPresenter (Extensions)
+
+private extension RootViewPresenter {
+    func showReaderTeam(named teamName: String) {
+        let topic = ContextManager.shared.mainContext.firstObject(
+            ofType: ReaderTeamTopic.self,
+            matching: NSPredicate(format: "slug = %@", teamName)
+        )
+        if let topic {
+            showReader(path: .topic(topic))
+        }
+    }
+
+    func showReaderList(named listName: String, forUser user: String) {
+        let context = ContextManager.sharedInstance().mainContext
+        if let topic = ReaderListTopic.named(listName, forUser: user, in: context) {
+            showReader(path: .topic(topic))
+        }
+    }
+
+    /// - warning: This method performs the navigation asyncronously after
+    /// fetching the information about the stream from the backend.
+    func showReaderStream(with siteID: Int, isFeed: Bool) {
+        getSiteTopic(siteID: NSNumber(value: siteID), isFeed: isFeed) { [weak self] result in
+            guard let topic = try? result.get() else { return }
+            self?.showReader(path: .topic(topic))
+        }
+    }
+
+    private func getSiteTopic(siteID: NSNumber, isFeed: Bool, completion: @escaping (Result<ReaderSiteTopic, Error>) -> Void) {
+        let service = ReaderTopicService(coreDataStack: ContextManager.shared)
+        service.siteTopicForSite(withID: siteID, isFeed: isFeed, success: { objectID, isFollowing in
+            guard let objectID = objectID,
+                  let topic = try? ContextManager.sharedInstance().mainContext.existingObject(with: objectID) as? ReaderSiteTopic else {
+                DDLogError("Reader: Error retriving site topic - invalid Site Id")
+                return
```

Comment:
Fixed. It doesn't really affect anything though because the code calling it ignores errors and doesn't wait for a completion.

I had a universal link handy to re-rest it to make sure it still works after the changes:

https://github.com/user-attachments/assets/f5b6e161-e342-4168-9208-968e8325e35a